src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,100 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * Handles all User Interface of the program.
+ */
+public class Ui {
+
+    private String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+    public Ui() {
+
+    }
+
+    /**
+     * Display the welcome message to the user.
+     */
+    public void showWelcome() {
+        System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?");
+    }
+
+    /**
+     * Display a prompt asking user for an action.
+     */
+    public void showLoopWelcome() {
+        System.out.println("What else can I do for you?");
+    }
+
+    /**
+     * Reads input by user and returns what user have typed.
+     *
+     * @return A String written by user.
+     */
+    public String readCommand() {
+        Scanner scanner = new Scanner(System.in);
+        String input = "";
+        try {
+            input = scanner.nextLine();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        return input.trim();
+    }
+
+    /**
+     * Prints out an error message.
+     *
+     * @param msg The error message that is to be printed.
+     */
+    public void printErrorMessage(String msg) {
+        System.out.println(msg);
+    }
+
+    /**
+     * Display loading error message.
+     */
+    public void showLoadingError() {
+        System.out.println("Initialisation Error! We will override with a fresh state.");
+    }
+
+    /**
+     * Display goodbye message.
+     */
+    public void goodBye() {
```

Comment:
Should this method be renamed showGoodBye() to be consistent with the showWelcome() method?

Reply:
yeap good catch! I have made changes to it already. Thank you.